plugins/amazonq/shared/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/project/ProjectContextProvider.kt

```diff
@@ -53,25 +55,12 @@ class ProjectContextProvider(val project: Project, private val encoderServer: En
             }
         }
     }
-    data class IndexRequestPayload(
-        val filePaths: List<String>,
-        val projectRoot: String,
-        val refresh: Boolean,
-    )
 
     data class FileCollectionResult(
         val files: List<String>,
         val fileSize: Int,
     )
 
-    data class QueryRequestPayload(
-        val query: String,
-    )
-
-    data class UpdateIndexRequestPayload(
-        val filePath: String,
-    )
```

Comment:
move to https://github.com/aws/aws-toolkit-jetbrains/pull/4988/files#diff-00c500185316c5fcdd5fdf6196f00012ecfd3abb9fa9b0851c773d92a2e6e796R38-R44